pkg/virtctl/virtctl.go

```diff
@@ -35,6 +35,20 @@ type App interface {
 type Options struct {
 }
 
+func GetKubeConfig(flags *flag.FlagSet) string {
+	kubeConfig, err := flags.GetString("kubeconfig")
+	if kubeConfig == "" || err != nil {
+		kubeConfig := os.Getenv("kubeconfig")
+		if kubeConfig == "" {
+			kubeConfig = os.Getenv("KUBECONFIG")
+		}
+		if kubeConfig == "" {
+			kubeConfig = os.Getenv("HOME") + "./kube/config"
```

Comment:
I think we should be using the os/user package here to get the home directory instead of the HOME env var. 

user.Current() will return the current user, and that struct has info about the home dir for the user.
https://golang.org/pkg/os/user/#Current